src/scf/self_consistent_field.jl

```diff
@@ -1,3 +1,5 @@
+using Plots
```

Comment:
Do we really want this? It'll massively increase load times...

Reply:
Or use requires?

Reply:
Crime already committed: https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/postprocess/band_structure.jl#L2

Reply:
Well that might be why it's so slow... Use requires there as well maybe?

Reply:
`using Plots` is 7s here (1.5beta)

Reply:
I thought about that, but the problem is that then you cannot plot bandstructures unless issue a `using Plots` first, which is a little counterintuitive.

Reply:
I'd say 7s of load time in most cases is worth a bit of extra boilerplate. Ideally we could revisit this once (if?) load times becomes less annoying

Reply:
Hmmm I have plots in my sysimage so for me it loads in 0.2 seconds ^^. No but ok I get the idea. I'll do that in a separate PR, though.